### ISSUE TITLE
This module will be work react js also ? 

### ISSUE THREAD

**Issue by siamraj** [morcmarc/react-native-cognito#7]
Actually we are try to do social authentication using react js. But we don't have find correct module for reactJS till now. My question is React native module will be work on react js project ?

**Comment by tobiastornros** [comment]
No, you should use https://github.com/aws/aws-amplify/tree/master/packages/amazon-cognito-identity-js with Reactjs
